setup-mini.cfg

```diff
@@ -105,7 +105,7 @@ install_requires =
     pymaven_patch >= 0.2.8
     requests >= 2.7.0
     saneyaml >= 0.6.0
-    spdx_tools == 0.7.0rc0
+    spdx_tools == 0.8.0rc2
```

Comment:
Thanks for the PR.
I had a question on this being a release-candidate.

The readme says the following:
```
The main features of v0.8 are:

    experimental support for the upcoming SPDX v3 specification (note, however, that support is neither complete nor stable at this point, as the spec is still evolving)
```

SPDX v3 is not out yet and the spec could change/evolve more from this point, and
it's admittedly `neither complete nor stable`, shouldn't we wait for a stable release of 
`spdx-tools` then?
(which I'm assuming would be after SPDX v3 would be out officially) 
Does this PR change the spec version of the SPDX output produced?

Reply:
Thanks for voicing your concerns, @AyanSinhaMahapatra, but I can alleviate them! :)
This PR does not introduce any SPDX v3 features into scancode and makes no use of the experimental/unstable SPDX-3 code.
We will release 0.8.0 soon and I'll update the dependency then. The 0.8.0 version is stable for SPDX v2, which is the only part of the library used in scancode.

So, to answer your last question:
No, this PR does not change the spec version of the SPDX output produced.